torch/csrc/jit/codegen/cuda/lower_validation.cpp

```diff
@@ -27,16 +24,12 @@ void IRValidate(Fusion* fusion) {
               ".");
         }
       }
-    } // if ir_utils::isTV
```

Comment:
why?

Reply:
Why was it needed in the first place?

This is the only place we do it, so it's inconsistent. Besides, are you confident that future modifications would correctly duplicate the conditions in the comments?